src/main/java/org/isf/patient/manager/PatientBrowserManager.java

```diff
@@ -391,7 +391,7 @@ protected void validate(Patient patient) throws OHDataValidationException{
     }
 
     private boolean checkAge(Patient patient) {
-	    Date now = new Date();
+	    Date now = new Date((new java.util.Date()).getTime());
```

Comment:
this is not changing, maybe you mean `Date now = new Date((new java.sql.Date()).getTime());` ?